chef/cookbooks/provisioner/libraries/repositories.rb

```diff
@@ -101,6 +127,8 @@ def get_repos(provisioner_server_node, platform, version)
             )
           when "12.0"
             repo_names = %w(
+              SUSE-Storage-1.0-Pool
+              SUSE-Storage-1.0-Updates
```

Comment:
As per my previous discussions, I thought this was the requirement. I will talk to Dirk once.